community/knot-resolver/APKBUILD

```diff
@@ -33,15 +33,30 @@ source="https://secure.nic.cz/files/$pkgname/$pkgname-$pkgver.tar.xz
 _flags="PREFIX=/usr
 	ETCDIR=/etc/$pkgname"
 
+prepare() {
+	default_prepare
+
+	cd "$builddir"
+	# Make sure embedded LMDB library is not used.
+	rm -rf contrib/lmdb
+}
+
+
 build() {
+	cd "$builddir"
```

Comment:
Why do you want to put these back? There is no stable branch that contains `community/knot-resolver` and it does not make sense to backport changes to `testing/knot-resolver`.

Reply:
Putting them back is going to prevent merge conflicts.

Reply:
Merge conflicts where exactly? There is nothing to merge into, this package is not in a stable branch / releaase yet, so there is nothing to backport.

Reply:
You were right. I was thinking about a branch I already closed out. I'll rebase and remove the `cd` lines later.